src/App.js

```diff
@@ -21,20 +21,29 @@ class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      authUser: false
+      authUser: false,
+      loggedIn: false
     };
   }
 
   componentDidMount = async () => {
     ipcRenderer.on("log-in-app", (e, msg) => {
-      console.log("inside loginapp");
       if (msg.error) {
         return console.log(msg.error);
       }
       this.setState({
         authUser: true
       });
     });
+
+    ipcRenderer.on("sign-up", (e, msg) => {
+      if (msg.error) {
+        return console.log(msg.error);
```

Comment:
@aviquez96 
We should provide feedback to the user indicating that there was an error when logging in. Perhaps an Alert(msg.error) will provide us with a basic feedback system for now?

Let me know!

Reply:
Hey @febg, I think I could implement that, no problem. 

On a similar note, the way our signup works right now is as follows: we have the "register" button disabled until all the parameters are filled in. That is, we check for all the fields to have something, and then the button will be enabled after that. The logic is this:

const isInvalid = organization === "" || username === "" || email === "" || password === "";
...
                   <Button
                    ...
                    disabled={isInvalid}
                    ...
                   >
                    Register
                  </Button>

On top of that, if the email doesn't follow the email format (have an '@'), it will always through an error. I couldn't think of anything super strict for username and organization, and as of now, password only checks if empty. This logic could be improved to have like 8 characters min, at least one symbol, etc. 

If you feel like, I could implement the logic for the password field, but besides that, anything else to change?

Synthesis: a) add alert(err) to ipcRender.on('sign-up') and b) update password logic

Reply:
@aviquez96 
`a) add alert(err) to ipcRender.on('sign-up') and b) update password logic`
 This will be ok for this PR, we can think about further user requirements later, since those will most likely come from management team